server/src/route/module.route.js

```diff
@@ -1,8 +1,10 @@
 const modules = require('../controller/module.controller');
+const checkAuth = require('../middleware/check-auth');
 
 module.exports = (app) => {
   app.get('/module', modules.findAll);
-  app.post('/module', modules.create);
-  app.delete('/module/:id', modules.destroy);
-  app.patch('/module/:id', modules.update);
+  app.post('/module', checkAuth, modules.create);
+  app.delete('/module/:id', checkAuth, modules.destroy);
+  app.patch('/module/:id', checkAuth, modules.update);
+  app.put('/module/:id', modules.reset);
```

Comment:
Having a public route for this implies that when one person resets a module, it is reset for all other visitors.
Why reset modules server side at all?

Reply:
That was the idea.
I made a separate route and made it public so that anyone can rest the steps
It shouldn't be that way!!